XogarWinGui/MainWindow.xaml.cs

```diff
@@ -39,20 +49,24 @@ public MainWindow()
                 }
             }
 
+            btnPlayRandom.Content = PLAY_RANDOM_TEXT;
+
             gamesLists = Playlists.Load();
             DataContext = this;
 
-            playlistBox.ItemsSource = playlistDictionary;
+            playlistBox.ItemsSource = AllPlaylists;
+            playlistBox.SelectedIndex = 0;
         }
 
         private void ReadInstallLocationAndLoadGames()
         {
-            FileStream reader = new FileStream(steamInstallDirStoragePath, FileMode.Open);
-            StreamReader locationReader = new StreamReader(reader);
-
-            String actualInstallDir = locationReader.ReadToEnd();
-            picker = new Games(actualInstallDir);
-            locationReader.Close();
+            using (FileStream reader = new FileStream(steamInstallDirStoragePath, FileMode.Open))
```

Comment:
This formatting is idiomatic C#; see [here](https://stackoverflow.com/questions/1329739/nested-using-statements-in-c-sharp) or [here](https://stackoverflow.com/questions/9396064/using-statement-with-multiple-variables)

However, it's your project, so if you'd still like I will change it.

Reply:
Curly braces is one of my pet peeves. I've been bitten by not using curly braces when they weren't required, seen code poorly maintained over a decade with various bugs because of lack of curly braces, and generally prefer using syntax less likely to allow bugs.

Similarly, I don't like Python because I've seen how poorly whitespace can be maintained over years.